### PR TITLE
[Ready for review] (early) September newsletter

### PR DESCRIPTION
**Pull request by KirstieJane**
### Summary

Doesn't fix an open issue

### List of changes proposed in this PR (pull-request)

* Added picture for newsletter
* Wrote newsletter to send in early September via tinyletter

### What should a reviewer concentrate their feedback on?

- [ ] Content ok? I know we could add more but I'm trying to go for done rather than perfect 😬 
- [ ] Any typos, mistakes in the information?
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: communications/newsletters/newsletter_06_Sept2019.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Everything is [CC-BY licenced](https://creativecommons.org/licenses/by/4.0) and available in `jpg` and `svg` format.
+We'd love to know which ones you like the best, and to receive any pictures of them in action in your own presentations.
+
+If you adapt any of them (for example by changing the colours) please send us a copy so we can add it to the repository to make it available for others in the future.
+
+![](images/ComboScriberia_ForNewsletter.png)
+
+## Turing Way talks
+
+One of Kirstie's tasks now that its September is to update the GitHub repsitory with lots of the great talks and presentations that happened over the summer.
```
Comment (rainsworth) [inline communications/newsletters/newsletter_06_Sept2019.md]:
```suggestion
One of Kirstie's tasks now that it's September is to update the GitHub repsitory with lots of the great talks and presentations that happened over the summer.
```

File: communications/newsletters/newsletter_06_Sept2019.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## New jobs
+
+Congratulations to three of the core _Turing Way_ team members who have started new jobs this summer.
+
+* [Rachael Ainsworth](https://twitter.com/rachaelevelyn) is now the Research Software Community Manager for the [Software Sustainability Institute](https://twitter.com/SoftwareSaved), based in Manchester.
+* [Patricia Herterich](https://twitter.com/PHerterich) has joined [Digital Curation](http://www.dcc.ac.uk/) to work on the [FAIRsFAIR](https://www.fairsfair.eu/) (Fostering Fair Data Practices in Europe) project in Edinburgh.
+* [Rosie Higman](https://twitter.com/RosieHLib) is the Research Data Manager for the [University of Sheffield Library](https://www.sheffield.ac.uk/library).
+
+We're so incredibly proud of all the Turing Way community members, and these roles in particular are incredibly important to the success of our goal of making "reproducible research too easy not to do".
+Congratulations Rachael, Patricia and Rosie!
```
Comment (rainsworth) [inline communications/newsletters/newsletter_06_Sept2019.md]: ❤️

**Comment by KirstieJane** [comment]
Since you're online @rainsworth, lets go ahead and add the Open Science Fair information. Can you ping me a sentence? I'll add it to the IRL section at the end! Thank you for reviewing 💖

**Comment by pherterich** [comment]
<3 What Rachael said about the Open Science Fair & plenty of other talks you are doing, but that can also be listed later this month when all the talks are up and linked and you can do a little report on how the collaboration cafes are going?

**Comment by KirstieJane** [comment]
Aaaah - you're all the best. So sorry for making you rush these reviews. I'll add Open Science Fair, @sgibson91 at RSE conf....anything else imminent that should go in? (My talks can go in a later one).

**Comment by rainsworth** [comment]
Feel free to adapt however you want, based it off the previous newsletter format:

"Rachael Ainsworth will present *The Turing Way* at the [Open Science Fair 2019](https://www.opensciencefair.eu/) in Porto, September 16-18 as a [poster](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/OSF19_Poster_submission.md) and during the [demo sessions](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/OSF19_Demo_submission.md)."

File: communications/newsletters/newsletter_06_Sept2019.md (hunk 1 above)
Comment (KirstieJane) [reply communications/newsletters/newsletter_06_Sept2019.md]: Great catch - thank you. I fixed it locally (to explain why I didn't click "commit suggestion")

**Comment by KirstieJane** [comment]
Thank you for the comments @rainsworth @pherterich and @malvikasharan! I think I've addressed them if you have time to take another look.

**Pull request merged**